src/draw/shapes.typ

```diff
@@ -674,6 +674,125 @@
   },)
 }
 
+
+/// Draws a n-pointed star.
+///
+/// ```typc example
+/// n-star((0,0), 5)
+///
+/// // An 8-pointed star, rotated
+/// n-star((2,0), 8, angle: 11.25deg)
+///
+/// // A 6-pointed star showing its inner hexagon
+/// n-star((4,0), 6, show-inner: true, style: red)
+/// ``` 
+///
+/// - origin (coordinate): Coordinate to draw the star's center at.
+/// - sides (int): Number of points of the star (>= 3).
+/// - angle (angle) = 0deg: Angle to rotate the star around its origin.
+/// - name (none, str): An optional name to identify the shape.
+///
+/// ## Styling
+/// Root: nstar
+/// - inner-radius (number) = 0.5: The radius of the circle on which the inner points lie.
+/// - outer-radius (number) = 1: The radius of the circle on which the outer points lie.
```

Comment:
Radius seems to be a single array `(inner,outer)` but the docstring specifies two distinct arguments.